scripts/iOS.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+set -o pipefail && \
+  xcodebuild clean build test \
+  -project BonMot.xcodeproj \
+  -scheme BonMot-iOS \
+  -sdk iphonesimulator \
+  -destination "id=E8DD285C-51EE-4DB5-B326-7E927686EC36" \
```

Comment:
#? why IDs? It seems like this will be super fragile - every time the simulators are regenerated, the IDs will be different. I don't think they're deterministic, are they? What was wrong with using names like on Travis?